pages/research_software/three_tier_view.md

```diff
@@ -4,16 +4,10 @@ title: Three-Tier Model of Research Software
 
 The **three-tier model of research software** provides a framework for understanding the diverse landscape of software 
 in research - from scripts, code, notebooks, to computational workflows, libraries, modules, frameworks, utilities and applications. 
```

Comment:
```suggestion
in research - from scripts, code, notebooks, to computational workflows, libraries, modules, frameworks, utilities and applications. This model distinguishes three tiers of research software, based on the intent of the research software - i.e. focussing on the purpose for which the software is being built:
```

Reply:
Oh will add this comment separately - looks like I missed it the first time.